recipe/meta.yaml

```diff
@@ -53,9 +53,9 @@ requirements:
     - python
     - future  # used in the generated python wrappers
     # VTK Third Party dependencies
-    - zlib 1.2.11
-    - freetype 2.8.1
-    - hdf5 1.10.1
+    - zlib 1.2.11.*
+    - freetype 2.8.1.*
+    - hdf5 1.10.1.*
```

Comment:
since we took out the   -DVTK_USE_SYSTEM_HDF5:BOOL=ON  above, do we still need hdf5?

Reply:
I think we do - netcdf seems to be always on and that needs hdf5. I think there is a problem with system htf5 with the new VTK - I get an undefined symbol there. I think there has been work on updating third parties which might cause it.

Reply:
I am confused, if you took out the VTK_USE_SYSTEM_HDF5, VTK will use its own mangled HDF5 or did I miss something?

Reply:
Yes, that is right.

Reply:
ah, okay. thanks.